src/Footer.tsx

```diff
@@ -1,22 +1,20 @@
 import { Box } from '@chakra-ui/layout';
-import { Text } from '@chakra-ui/react';
-import { useActor } from '@xstate/react';
+import { Link } from '@chakra-ui/react';
 import React from 'react';
-import { useSimulation } from './SimulationContext';
 
 export const Footer: React.FC = () => {
-  const sim = useSimulation();
-  const [state] = useActor(sim);
-
   return (
-    <Box padding="1" paddingInline="2" background="gray.900" gridArea="footer">
-      <Text
-        fontSize="sm"
-        title={JSON.stringify(state.value, null, 2)}
-        color="whiteAlpha.800"
-      >
-        {Array.from(state.tags).join(', ') ?? '&nbsp;'}
-      </Text>
+    <Box
```

Comment:
IMHO this currently blends with the panels on the right too much. It's also quite big - and quite annoying for me (as a user) because it distracts me. If our app would be mobile-friendly using a footer for this would probably not be an option due to space-limited environment. 

I feel like using a footer for this is a good fit for websites because the footer is not always visible. We have a full-screen application and having this visible at all times is just a waste of space (and a distraction, in my opinion). Especially that this is not something that people need at all times - most won't look into it at all. I've looked through a couple of apps and it seems like putting this in a sidebar is a common practice - it still seems to be very accessible this way, and not too hidden.

Reply:
You all really don't like footers 😂  👣  🚫 

Fine, I'll get rid of it.